### ISSUE TITLE
How to configure fractions in PeptideShaker

### ISSUE THREAD

**Issue by kazbek966** [compomics/searchgui#361]
There was similar issue already raised in 2020 however no instructions were provided. 
In particular the question is how to organize/group individually analyzed files into new fractions/groups that could be assigned to biological or technical replicates, digests or any other category. 

Currently, files are equaled with fractions and this is far from ideal situation. 

Thanks

**Comment by hbarsnes** [comment]
I agree that treating each file as a single fraction is not ideal. However, we sadly not not have the manpower to implement new features at the moment. But we’ll add your suggestion to the list of requested features and may get back to it sometime in the future.
